### PR TITLE
Proposal: Usability Testing (#15)

### PR DESCRIPTION
**Pull request by gregoryjschmidt**

### CODE REVIEW

File: 0015-usability-testing.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+## Usability Testing
+- Start Date: 2019/09/10
+- RFC PR:
```
Comment (internettrans) [inline 0015-usability-testing.md]:
```suggestion
- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/15
```

File: 0015-usability-testing.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	
+	
+## Alternatives
+**No user testing** this may be faster to move designs into the software development stage, but is high risk for requiring revision later on.
+
+**Quantitative User Testing** is based on observations from a larger group of participants. Example: card sorting, tree testing, a/b testing. The goal is to identify quantitatively which design is optimal and preferred. These methods are valuable, but generally more labour intensive. This is a different type of user testing that that of qualitative testing proposed in this RFP.
+
+**Focus Groups** are distinctly different from user testing. Focus groups ask users what they may like to see or do. User testing involves watching users actually use something. Focus groups are not useful for the testing phase of UX work. They also have limited in the Discovery phase value https://medium.com/mule-design/focus-groups-are-worthless-7d30891e58f1
+
+**Accesibility Evaluation** is testing a design for accesibility across different user abilities. This is the subject for a different RFC.
```
Comment (internettrans) [inline 0015-usability-testing.md]:
```suggestion
**Accessibility Evaluation** is testing a design for accessibility across different user abilities. This is the subject for a different RFC.
```

**Review by internettrans — COMMENTED** [review]
👏 👍 

This is good content, thanks for putting it together. What is outlined in this RFC is a departure from how previous things have been built at OpenMRS I think -- this will benefit OpenMRS' users greatly.

I'm not sure exactly if/where the following would go in this RFC, but here are a few things I've learned / found useful when doing user testing. I read through some of the articles that you linked to and they mentioned some of them there already, but thought I'd share:

- "Don't lead the witness": The way you phrase questions has a big impact on how they are answered. Often people will answer a question knowing how you want it to be answered
- "Is this good? Would you use it?" are usually unreliable questions
- "Do the user testing with the people who actually use it": at my previous job, we did too much user testing with employees of the company who had an accounting background instead of with customers who were accountants paying for the software. People who use the software have "skin in the game" and are much less likely to be satisfied with fluffy descriptions and mockups of features. They often know what actually matters and tell you it bluntly 😄 
 - "Invite developers, product managers, and other stakeholders to user testing": this was very successful at my last two jobs in building group consensus on product designs. People might not always be able to come, but it's worth it to invite them.
- "Lots of lukewarm yes'es is a really bad sign": if all five people give positive feedback without anything constructively critical, then that might mean we're solving the wrong problem, are talking to the wrong users, or that the UX didn't really click with them.

I left a couple small typo-ish suggestions on the RFC text itself, but am ready to approve this after those small things are cleaned up.

**Comment by gregoryjschmidt** [comment]
Thanks for the feedback - and fixing the typos

I was thinking of building in the Wiki a more detailed, step-by-step process on usability testing (still under development). And keeping this RFC short by focusing on the need to do usability testing, but not go in the RFC into too many details of how to do it.

Was thinking of adding your advice on this page, https://wiki.openmrs.org/display/projects/Advice+from+the+field++%7C++Usability+Testing

**Review by fatmali — APPROVED** [review]

**Review by internettrans — APPROVED** [review]

**Comment by internettrans** [comment]
One more approval and we will merge. @brandones @angshu?

**Review by brandones — APPROVED** [review]
Yeah, looks good!

**Pull request merged**